TWLight/applications/tests.py

```diff
@@ -3409,6 +3409,27 @@ def test_no_status_form_for_bundle_partners(self):
         # Not even for coordinators
         self.assertNotContains(response, 'name="status"')
 
+        def test_more_applications_than_accounts_available_for_proxy(self):
+            self.partner.authorization_method = Partner.PROXY
+            self.partner.accounts_available = 1
+            self.partner.save()
+            self.user.set_password("editor")
+            self.user.save()
+            self.user.userprofile.terms_of_use = True
+            self.user.userprofile.save()
+
+            self.client.login(username=self.user.username, password="editor")
+            self.message_patcher.stop()
+            response = self.client.post(self.url)
+            messages = list(response.context["messages"])
+            self.assertEqual(len(messages), 1)
+            self.assertEqual(
```

Comment:
This test is pretty brittle since it depends on the entirety of the message staying the same. Instead of testing the final output of the view, you could just test the true/false response of the helper function by feeding it a few apps and testing for true/false output.

Reply:
Makes sense. Fixed!